frontend/src/components/MoveSelector/MoveSelector.tsx

```diff
@@ -53,12 +53,10 @@ export const MoveSelector: React.FunctionComponent<MoveSelectorProps> = ({
     if (gameData?.isRoundInProgress) {
       setDisabled(false);
     }
-  },
-    [gameData?.isRoundInProgress]
-  )
+  }, [gameData?.isRoundInProgress]);
 
-  const handleClick = (e: React.MouseEvent, selection: string) => {
-    e.stopPropagation();
+  const handleClick = (selection: string) => {
+    console.log(ClientEvents.SET_MOVE, selection);
```

Comment:
Remove `console.log`:

```suggestion

```

Would you elaborate on the reason why removing the propagation would be a better option, please?

Reply:
I wanted to ask you the reason why you need stop propagation actually, it doesn't seem needed to me. Did you have any side effect due to event propagation? Trying it out quickly didn't seem to effect the curse of the game